frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -119,24 +178,28 @@ const validationConfig: VMSettingsValidationConfig = {
           ]
         : [VMWizardProps.activeNamespace, VMWizardProps.virtualMachines];
     },
-    validator: validateVm,
+    validators: [validateVm],
```

Comment:
I think it's better to have multiple validators like this. It is clearer that a field has multiple validators rather than a validator that may or may not branch into multiple validators inside it.

Reply:
Why is it better? 
It is unclear what the execution of the validators should be e.g. the order and the validation message.

IMO the interface should be expressive enough for future changes (possible cross validation).